### PR TITLE
feat: Add WebView sample for macOS

### LINKED ISSUE #151: macOS WebView sample is missing
## What would you like to be added:

WebView sample on macOS

## Why is this needed:

🚀

## For which Platform:

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly Renderers for Xamarin.Forms
- [ ] Windows
- [x] macOS

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): fixes #151

## PR Type

What kind of change does this PR introduce?

- Feature

## What is the current behavior?

macOS WebView not in samples

## What is the new behavior?

macOS WebView in samples

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested on iOS.
- [ ] Tested on Wasm.
- [ ] Tested on Android.
- [ ] Tested on UWP.
- [x] Tested in both **Light** and **Dark** themes.
- [x] Associated with an issue (GitHub or internal)

### CODE REVIEW

File: Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/WebViewSamplePage.xaml.cs [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 namespace Uno.Gallery.Views.Samples
 {
-	// There is no WebView implementation in WebAssembly and MacOS
-#if !__WASM__ && !__MACOS__
+	// There is no WebView implementation in WebAssembly
+#if !__WASM__
```
Comment (carldebilly) [inline Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/WebViewSamplePage.xaml.cs:6]: Is it activated for SKIA?

**Review by carldebilly — APPROVED** [review]
I supposed it's working? ;-)

File: Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/WebViewSamplePage.xaml.cs (hunk 1 above)
Comment (MartinZikmund) [reply Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/WebViewSamplePage.xaml.cs]: @carldebilly It should not be 😮 ! I disabled the sample there.

**Review by carldebilly — APPROVED** [review]

**Review by Xiaoy312 — APPROVED** [review]
might want to adjust the comment above as well

**Pull request merged**